skimage/transform/tests/test_pyramids.py

```diff
@@ -135,3 +135,13 @@ def test_check_factor():
         pyramids._check_factor(0.99)
     with testing.raises(ValueError):
         pyramids._check_factor(- 2)
+
+
+@pytest.mark.parametrize('dtype, expected',
+                         zip(['float32', 'float64', 'uint8', 'int64'],
+                             ['float32', 'float64', 'float64', 'float64']))
```

Comment:
A list of tuples would have been clearer.

Reply:
I hesitated =)